LitvischenkoYevhen/HW6/HW6_task_1.py

```diff
@@ -0,0 +1,3 @@
+print('Even numbers that are divisible by 2 :', [x for x in range(1, 11) if x%2 == 0])
```

Comment:
look at this code
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')


output of your program in the second case must be---- [3, 9] is an odd multiple of 3 
without  6

Reply:
I didn't read the task carefully. thankyou.
Should I fix this? or is it not necessary?
i dont know how its make. branch is merged alredy.